dashboard/v1.1/src/pages/Config/Config.tsx

```diff
@@ -84,8 +85,10 @@ const Config: FC<{}> = () => {
           }
         });
       });
+      var routename =
```

Comment:
`var` should be `let` if it's being updated. Else, please go with `const`.

BTW, I don't see `routename` being updated anywhere, so please prefer `const`.